source/core/tests/integration/gold/application/streams/test_migrated_transactions.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+
+from pyspark.sql import SparkSession
+from pytest_mock import MockerFixture
+
+import tests.helpers.table_helper as table_helper
+from core.bronze.infrastructure.config.table_names import TableNames as BronzeTableNames
+from core.gold.application.streams import migrated_transactions_stream as mit
+from core.gold.infrastructure.config import GoldTableNames
+from core.settings.bronze_settings import BronzeSettings
+from core.settings.gold_settings import GoldSettings
+from tests.helpers.builders.migrated_transactions_builder import MigratedTransactionsBuilder
+
+
+def test__migrated_transactions__should_save_in_gold_measurements(
+    mock_checkpoint_path, spark: SparkSession, migrations_executed, mocker: MockerFixture
+) -> None:
+    # Arrange
+    mocker.patch(f"{mit.__name__}.spark_session.initialize_spark", return_value=spark)
+    bronze_settings = BronzeSettings()
+    gold_settings = GoldSettings()
+
+    expected_transaction_id = "UnitTestingMigratedStream"
```

Comment:
```suggestion
    expected_transaction_id = identifier_helper.create_random_id()
```

Or something like that. We do have a `identifier_helper.py` to create a random strings to avoid having hardcoded values for tests :D